context.item.extras/addon.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="context.item.extras" name="Extras" version="1.3.2" provider-name="takoi,pkscout">
+  <requires>
+    <import addon="xbmc.python" version="2.26.0"/>
+	<import addon="script.module.kodi-six" version="0.0.3" />
+    <import addon="script.module.routing" version="0.2.0"/>
+  </requires>
+  <extension point="kodi.context.item">
+    <menu id="kodi.core.main">
+      <item library="addon.py">
+        <label>30000</label>
+        <visible>String.IsEqual(ListItem.dbtype,movie)
+            | String.IsEqual(ListItem.dbtype,tvshow)
+            | String.IsEqual(ListItem.dbtype,season)
+        </visible>
+      </item>
+    </menu>
+  </extension>
+  <extension point="xbmc.python.pluginsource" library="plugin.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="kodi.addon.metadata">
+    <news>- Kodi v19 support.</news>
+    <platform>all</platform>
+    <summary lang="en_GB">View extras and special features</summary>
+    <description lang="en_GB">Provides an easy way to browse and view movie and TV show extras. Extras can be accessed from the context menu in the video library. By default, the add-on will look in the "Extras" sub-folder for content (can be changed in settings).
+
+Tip: for how to avoid extras beings scraped to library, see http://kodi.wiki/view/Add-on:Extras</description>
+    <license>GPL-3.0-ONLY</license>
+    <forum>https://forum.kodi.tv/showthread.php?tid=312338</forum>
+    <source>https://github.com/tamland/kodi-addon-extras</source>
```

Comment:
Please change upstream

Reply:
I can, but there was some indication when I emailed tamland that they might want to start doing updates again in the future. I have collaborator access to the repo that is referenced, so once this all got approved I was going to push the changes there. I only used a repo on my account so I could submit it to Codacy for review.